Tiltfile

```diff
@@ -39,12 +39,20 @@ for role in roles:
 if len(roles_rules_mapping["ClusterRole"]) == 0 or len(roles_rules_mapping["Role"]) == 0:
     fail("Failed to load cluster and namespace roles")
 
+additionalHelmInstallationArgs = {}
```

Comment:
nit: let's use snake case for variable names https://peps.python.org/pep-0008/#function-and-variable-names